qa/automation/Starynina.Scenarios/3. Send CV/3.2.Send CV.feature

```diff
@@ -0,0 +1,136 @@
+	| email with .com (star@gmail.com)     | accepted                 | 
+	| email with .ru (star@mail.ru)        | accepted                 | 
+	| email with wordInCyrillic@gmail.com  | accepted                 | 
+
+@negative
+Scenario: 3.2_08 User enters email with more than maximum characters in the CV form 
+	When user enters name of 71 characters or more in the "Email" field
+	Then the error messages "The maximum number of characters is 70" for "Name" field is appeared
+	And the "Email" field should be highligted in red
+
+@negative
+Scenario: 3.2_09 User enters message with more than maximum characters in the CV form 
+	When user enters data of 501 characters or more in the "Message" field
+	Then the error messages "The maximum number of characters is 500" for "Message" field should appear
+	And the "Message" field should be highligted in red
+
+Scenario Outline: 3.2_10 User can't attach CV file of invalid file format
+	Given user clicks the "Attach CV" button
+	When user chooses file of <invalid format>
+	Then the error messages "Something went wrong Invalid file format. Please try PDF, DOC, DOCS or RTF" should appear
+
+	Examples:
+	| invalid format |
+	| JPEG           |
+	| TXT            |
+	| AVI            |
+
+Scenario: 3.2_11 User can't attach CV file of wrong file size
+	Given user clicks the "Attach CV" button
+	When user chooses file of valid DOC format 10.1 Mb size
+	Then the error messages "Something went wrong Max file size 10 Mb" is appeared
+
+Scenario: 3.2_12 User can fill the CV form with valid text data
+	Given user enters valid name of more than one words "Nina star" in the "Name" field
+	And user enters valid email "star@gmail.com" in the "Email" field
+	And user enters message "Hello" in the "Message" field
+	When user clicks at any place on the page
+	Then the "Name", "Email" and "Message" fields are highligted in green
+
+Scenario Outline: 3.2_13 User can send the CV form
+	Given user enters <valid name> in the "Name" field
+	And user enters <valid email> in the "Email" field
+	And user clicks the "Attach CV" button and attach CV of <valid format> and <valid size>
+	And user clicks the checkbox "I agree to the personal data processing" <checkbox>
+	When user clicks the "Send" button
+	Then the CV form is sent successfully and the message "Thanks for being awesome! We have received your message and will come back to you as soon as we process it." should appeared
+
+	Examples:
+	| valid name                       | valid email                         | valid format | valid size |
+	| two characterw (n s)             | email with .com (star@gmail.com)    | PDF          | 0.01 Mb    |
+	| two words in English (nina star) | email with .tu (star@mail.ru)       | DOC          | 5 Mb       |
+	| two words in Cyrillic            | email with wordInCyrillic@gmail.com | RTF          | 9.99 Mb    |
+
+Scenario Outline: 3.2_14 User can't leave required fields in the CV form empty
```

Comment:
Looks like incorrect formatting. 
And incorrect syntax, the case is not executed. 
re you sure that this case is your case? 
I found similar mistakes in another PR of your colleague.